### PR TITLE
Add post for Go OTel metric SDK reaching stable status

### PR DESCRIPTION
**Pull request by theletterf**
A simple notice announcing the stability of the Go Otel metric SDK.

### CODE REVIEW

**Review by svrnm — APPROVED** [review]

File: content/en/blog/2023/otel-go-metrics-sdk-stable.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: First stable release of OTel Go metric SDK
+linkTitle: Go Metric SDK is Stable
+date: 2023-09-29
+author: '[Fabrizio Ferri-Benedetti](https://github.com/theletterf/) (Splunk)'
+cSpell:ignore: Benedetti Fabrizio Ferri
+---
+
+The Go metric SDK has finally reached stable status with version 1.19.0 of
```
Comment (pellared) [inline content/en/blog/2023/otel-go-metrics-sdk-stable.md:9]: nit

```suggestion
The Go metrics SDK has finally reached stable status with version 1.19.0 of
```

Comment (pellared) [inline content/en/blog/2023/otel-go-metrics-sdk-stable.md:9]: Do we want to add a hyperlink to https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.19.0?

File: content/en/blog/2023/otel-go-metrics-sdk-stable.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+OpenTelemetry Go. Project stability guarantees now apply to the
+[go.opentelemetry.io/otel/sdk/metric](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/metric)
+package.
```
Comment (pellared) [inline content/en/blog/2023/otel-go-metrics-sdk-stable.md:12]:
```suggestion
module.
```

module (as there are more packages) 😉

File: content/en/blog/2023/otel-go-metrics-sdk-stable.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+To get started with OTel Go metrics,
+[read the documentation](/docs/instrumentation/go/manual) or browse the
+[examples](https://github.com/open-telemetry/opentelemetry-go/tree/main/example)
+for demonstrations of different instrumentation scenarios covered by the
+OpenTelemetry Go Instrumentation.
```
Comment (pellared) [inline content/en/blog/2023/otel-go-metrics-sdk-stable.md:18]: I think we can simply refer to these instead
- `/docs/instrumentation/go/getting-started`
- `/docs/instrumentation/go/manual#metrics`

E.g.
```suggestion
To learn more check the [getting started](/docs/instrumentation/go/getting-started)
and [manual metrics](/docs/instrumentation/go/manual#metrics) documentation.
```

The getting started has reference to examples.

**Review by MrAlias — APPROVED** [review]

**Review by pellared — APPROVED** [review]
:shipit:

**Comment by theletterf** [comment]
@svrnm I've updated the date, I think this one is ready for a merge.

**Pull request merged**